### PR TITLE
[docs] 7.0.0 release notes and breaking changes

### PR DESCRIPTION
**Pull request by bmorelli25**
* Adds in 7.0.0 server release notes (combined list of alpha, beta, RC)
* Moves APM Overview breaking changes:
  `/breaking-7.0.0-beta1.html` -> `/breaking-7.0.0.html`
* Replaces "notes" slug with "highlights" starting in 7.0.0 (APM release highlights)
* Removes "Change ownership to apm-server user/group for deb/rpm installs" from the release highlights page.
* Grammar/formatting

### CODE REVIEW

**Comment by bmorelli25** [comment]
@sqren do you (or anyone else) know if this line can be removed in 7.0:
> Be aware that this is a beta release. This script is **not** meant to be run on production data at this stage.

From here:
https://github.com/elastic/apm-server/blob/f83df3048bda61fc76d2a6a82c1f1d46f4835676/docs/guide/apm-breaking-changes.asciidoc#L30-L36

File: changelogs/7.0.asciidoc [hunk 1]
```diff
@@ -3,17 +3,66 @@
...
+- Remove `frontend` setting {pull}1751[1751].
+- Remove `metrics.enabled` setting {pull}1759[1759].
+- Remove dashboards from being shipped with APM Server and all logic around them {pull}[1815].
+
+[float]
+==== Bug fixes
+
+- Ensure setup cmd uses expected configuration {pull}1934[1934]. 
+- Ensure host.name is not added {pull}1934[1934], {pull}1982[1982].
+- Update Go version to 1.11.5 {pull}1840[1840], {pull}1950[1950].
```
Comment (jalvz) [inline changelogs/7.0.asciidoc]: hmm, how this is a bug fix? 
i'll also remove L39 (Update go to 1.11.2)

File: changelogs/7.0.asciidoc [hunk 2]
```diff
@@ -3,17 +3,66 @@
...
+
+* {apm-server-ref-v}/breaking-changes.html[Breaking changes in 7.0]
+
+[float]
+==== Added
+
+- Update Elastic APM Go agent to v1.1.2 {pull}1711[1711], {pull}1728[1728].
+- Update response format of the healthcheck handler and prettyfy all JSON responses {pull}1748[1748].
+- Add transaction.type to error data {pull}1781[1781].
+- Rename transaction.span_count.dropped.total to transaction.span_count.dropped {pull}1809[1809].
```
Comment (jalvz) [inline changelogs/7.0.asciidoc]: i don't know how it was reflected on previous releases or if it was discussed previously, but i'd say that L30, L31, L32, L35 and L36 are breaking changes

**Comment by sqren** [comment]
@bmorelli25 That's almost certainly outdated, but @graphaelli knows for sure.

**Comment by graphaelli** [comment]
Yes, that should be removed now, thanks for catching it.

File: changelogs/7.0.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply changelogs/7.0.asciidoc]: #1840 was a security issue, I think that's why it's in the bug fix section. I'll remove them.

File: changelogs/7.0.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply changelogs/7.0.asciidoc]: They were in the added column for `7.0.0-beta1`. Moved to breaking changes

**Pull request merged**